.github/workflows/macos.yml

```diff
@@ -30,18 +34,19 @@ jobs:
           sed -i.bak '/^AM_INIT_AUTOMAKE(\[-Wno-portability 1\.14\])$/s/14/11/' modules/oniguruma/configure.ac
       - name: Build
         env:
-          CC: ${{ matrix.compiler }}
+          CC: '${{ matrix.compiler }} -arch ${{ matrix.arch }}'
           MAKEVARS: ${{ matrix.makevars }}
         run: |
           autoreconf -fi
           rm src/lexer.c src/lexer.h
           rm src/parser.c src/parser.h
-          ./configure --disable-valgrind --with-oniguruma=builtin YACC="$(brew --prefix)/opt/bison/bin/bison -y" $COVERAGE
+          ./configure --disable-valgrind --with-oniguruma=builtin --host=${{ matrix.arch }}-apple-darwin$(uname -r) YACC="$(brew --prefix)/opt/bison/bin/bison -y" $COVERAGE
           make -j4
+          echo "$(file ./jq) built against host ${{ matrix.arch }}-apple-darwin$(uname -r)")
```

Comment:
I'm guessing the build error:
```
/Users/runner/work/_temp/7b085663-5c80-4857-8ebd-8f06e512b15e.sh: line 6: syntax error near unexpected token `)'
```
Is because of the ending ")"

Reply:
There you see what happens if you can't debug locally ;)

Reply:
Yeap is a mess to debug and interate :)